pandas/tests/io/test_sql.py

```diff
@@ -866,6 +866,12 @@ def test_get_schema(self):
         create_sql = sql.get_schema(self.test_frame1, "test", con=self.conn)
         assert "CREATE" in create_sql
 
+    def test_get_schema_with_schema(self):
```

Comment:
can you add the github issue number here, e.g. `# GH8624` (but with the correct number)